### PR TITLE
Added course component

### PR DESCRIPTION
**Pull request by Mav-Ivan**
image

### CODE REVIEW

File: src/components/course-card/CourseCard.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import IconButton from '@mui/material/IconButton'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import Divider from '@mui/material/Divider'
+
+import { styles } from '~/components/course-card/CourseCard.styles'
+import SubjectLevelChips from '../subject-level-chips/SubjectLevelChips'
```
Comment (TSlashDreamy) [inline src/components/course-card/CourseCard.tsx:10]: import this component like others, with `~`

File: src/components/course-card/CourseCard.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import { getFormattedDate } from '~/utils/helper-functions'
+import { Course } from '~/types'
+
+interface CourseCardProps {
+  course: Course
+}
+
+const CourseCard: FC<CourseCardProps> = ({ course }) => {
+  const {
+    _id,
```
Comment (TSlashDreamy) [inline src/components/course-card/CourseCard.tsx:20]: remove unused variable

File: src/components/course-card/CourseCard.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  const date = getFormattedDate({ date: createdAt })
+
+  return (
+    <Box sx={styles.card}>
+      <Box>
+        <Typography sx={styles.title} variant='h5'>
+          {title}
+        </Typography>
+        <Typography sx={styles.description} variant='body1'>
```
Comment (TSlashDreamy) [inline src/components/course-card/CourseCard.tsx:38]: maybe we can write `variant` in styles?

Comment (Mav-Ivan) [reply src/components/course-card/CourseCard.tsx]: yeah that`s better

File: src/components/course-card/CourseCard.tsx (hunk 2 above)
Comment (Mav-Ivan) [reply src/components/course-card/CourseCard.tsx]: done

File: src/components/course-card/CourseCard.tsx (hunk 1 above)
Comment (Mav-Ivan) [reply src/components/course-card/CourseCard.tsx]: done

**Review by TSlashDreamy — APPROVED** [review]

File: src/components/course-card/CourseCard.styles.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+export const styles = {
+  card: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    maxWidth: '312px',
+    minHeight: '245px',
+    backgroundColor: 'basic.white',
+    borderRadius: '6px',
+    p: '24px 24px 16px 24px'
```
Comment (Tolik170) [inline src/components/course-card/CourseCard.styles.ts:13]: p: '24px 24px 16px' should work too

File: src/components/course-card/CourseCard.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import IconButton from '@mui/material/IconButton'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import Divider from '@mui/material/Divider'
+
+import { styles } from '~/components/course-card/CourseCard.styles'
+import SubjectLevelChips from '~/components/subject-level-chips/SubjectLevelChips'
```
Comment (Tolik170) [inline src/components/course-card/CourseCard.tsx:10]: move component import above

**Review by OlyaKorchan — APPROVED** [review]

File: src/components/course-card/CourseCard.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+        <Typography sx={styles.title}>{title}</Typography>
+        <Typography sx={styles.description}>{description}</Typography>
+        <SubjectLevelChips
+          color={category.appearance.color}
+          proficiencyLevel={proficiencyLevel}
+          subject={subject.name}
+          sx={styles.chipContainer}
+        />
+        <Typography sx={styles.secondaryText}>
+          {`${sections.length} ${sections.length > 1 ? 'sections' : 'section'}`}
```
Comment (Tolik170) [inline src/components/course-card/CourseCard.tsx:43]: I think, this text should be in translation

**Review by Tolik170 — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

File: src/components/course-card/CourseCard.styles.ts (hunk 4 above)
Comment (Mav-Ivan) [reply src/components/course-card/CourseCard.styles.ts]: yeah you right

File: src/components/course-card/CourseCard.tsx (hunk 5 above)
Comment (Mav-Ivan) [reply src/components/course-card/CourseCard.tsx:8]: done

File: src/components/course-card/CourseCard.tsx (hunk 6 above)
Comment (Mav-Ivan) [reply src/components/course-card/CourseCard.tsx]: done

**Pull request merged**